server/adaptors/integrations/__data__/repository/aws_rds/schemas/aws_rds-1.0.0.mapping.json

```diff
@@ -0,0 +1,298 @@
+{
+  "template": {
+    "mappings": {
+      "_meta": {
+        "version": "1.0.0",
+        "catalog": "observability",
+        "type": "logs",
+        "component": "aws_vpc_flow"
+      },
+      "properties": {
+        "aws": {
+          "properties": {
+            "rds": {
+              "properties": {
+                "db-identifier": {
+                  "type": "text",
+                  "fields": {
+                    "keyword": {
+                      "type": "keyword",
+                      "ignore_above": 256
+                    }
+                  }
+                },
+                "sq-user": {
+                  "type": "text",
+                  "fields": {
+                    "keyword": {
+                      "type": "keyword",
+                      "ignore_above": 256
+                    }
+                  }
+                },
+                "sq-db-name": {
+                  "type": "text",
+                  "fields": {
+                    "keyword": {
+                      "type": "keyword",
+                      "ignore_above": 256
+                    }
+                  }
+                },
+                "sq-host-name": {
+                  "type": "text",
+                  "fields": {
+                    "keyword": {
+                      "type": "keyword",
+                      "ignore_above": 256
+                    }
+                  }
+                },
+                "sq-ip": {
+                  "type": "ip"
+                },
+                "sq-id": {
+                  "type": "text",
+                  "fields": {
+                    "keyword": {
+                      "type": "keyword",
+                      "ignore_above": 256
+                    }
+                  }
+                },
+                "sq-duration": {
+                  "type": "double"
+                },
+                "sq-lock-wait": {
+                  "type": "double"
+                },
+                "sq-rows-sent": {
+                  "type": "long"
+                },
+                "sq-rows-examined": {
+                  "type": "long"
+                },
+                "sq-table-name": {
+                  "type": "text",
```

Comment:
wondering here why these are mapped as text as opposed to keyword, any reason for doing so?